### PR TITLE
Link plant variety when adding new plant line in PP submission

### LINKED ISSUE #141: Add link to a chosen PlantVariety when user creates a new PlantLine
In the submission step where (and if) the user decides to create a new PL.

### PR DESCRIPTION
**Pull request by kammerer**
Fixes #141

### CODE REVIEW

File: app/services/submission/plant_population_finalizer.rb [hunk 1]
```diff
@@ -24,11 +24,12 @@ def create_new_plant_lines
     @new_plant_lines = (submission.content.step03.new_plant_lines || []).map do |attrs|
       attrs = attrs.with_indifferent_access
       taxonomy_term = TaxonomyTerm.find_by!(name: attrs.delete(:taxonomy_term))
+      plant_variety = PlantVariety.find_by!(plant_variety_name: attrs.delete(:plant_variety_name))
       attrs = attrs.merge(
         taxonomy_term_id: taxonomy_term.id,
+        plant_variety_id: plant_variety.id,
         date_entered: Date.today,
         data_provenance: submission.content.step04.data_provenance,
```
Comment (Nuanda) [inline app/services/submission/plant_population_finalizer.rb]: Extra comma at the end of the line.

By the way, can you also remove `comments: ''` from `create_plant_population_lists` at the end of this file?

Comment (kammerer) [inline app/services/submission/plant_population_finalizer.rb]: Some people say it is better to add these extra commas, both ways are fine with me.

**Comment by kammerer** [comment]
Updated PR

**Comment by Nuanda** [comment]
:+1:

**Pull request merged**